README.md

```diff
@@ -35,7 +35,11 @@ Replace `<VERSION>` with the version you want to install.
 ## Configure Parse Server
 
 ```js
-const PushAdapter = require('@parse/push-adapter').default;
+import { ParsePushAdapter } from '@parse/push-adapter';
+
+// For CommonJS replace the import statemtent above with the following line:
+// const PushAdapter = require('@parse/push-adapter').default;
+
 const parseServerOptions = {
   push: {
     adapter: new PushAdapter({
```

Comment:
This doesn't match the import above

Reply:
ES6 import imports the default export

Reply:
I see what you mean, fixed in https://github.com/parse-community/parse-server-push-adapter/pull/249